kerl

```diff
@@ -1922,9 +1923,22 @@ maybe_remove() {
 }
 
 list_print() {
-    if [ -f "$KERL_BASE_DIR/otp_$1" ]; then
-        if [ "$(\wc -l "$KERL_BASE_DIR/otp_$1")" != '0' ]; then
-            cat "$KERL_BASE_DIR/otp_$1"
+    list_type=$1 # releases | builds | installations
+    list=$KERL_BASE_DIR/otp_${list_type}
+    if [ -f "$list" ]; then
+        if [ "$(\wc -l "$list")" != '0' ]; then
+            if [ "${list_type}" = releases ]; then
+                awk -v target="$OLDEST_OTP_SUPPORTED" '
+                {
+                    split($0, v, ".")
+                    if (v[1] >= target && !/^R/) {
+                        printf "%s\n",$0
+                    }
+                }' "$list"
+            else
+                cat "$KERL_BASE_DIR/otp_$1"
+            fi
+
             return 0
         fi
     fi
```

Comment:
The goal is not to change internal behaviour (or the actual `releases` file) but rather just trim the list. A further pull request will remove dead code, effectively changing "the support".

In any case do lemme know if you think the generated `otp_releases` should also be trimmed (e.g. for consistency), in which case this code would be reverted but the code to create the file would be updated.